.maintain/monitoring/healthcheck-lambda.js

```diff
@@ -0,0 +1,185 @@
+const https = require('https');
+const http = require('http');
+const url = require('url');
+const webhookUrl = process.env.SLACK_NOTIFCATION_WEBHOOK_URL;
```

Comment:
I switched this and the node url JSON config to be env vars so we don't have to fiddle with the code when pushing out updates.